experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -30,16 +30,21 @@ export abstract class OTLPExporterBrowserBase<
   ServiceRequest,
 > extends OTLPExporterBase<OTLPExporterConfigBase, ExportItem, ServiceRequest> {
   protected _headers: Record<string, string>;
-  private _useXHR: boolean = false;
+  private sendMethod: 'beacon' | 'xhr' | 'fetch' = 'beacon';
```

Comment:
It would be "nicer" (from a size perspective) to have these as a const (numeric) enum value, rather than strings.

Reply:
OK. 
changed at da0c60774af9d43653451009b347cb60b31e2779